app/frontend/stores/notification-store.ts

```diff
@@ -75,6 +79,14 @@ export const NotificationStoreModel = types
             )}`,
           },
         ]
+      } else if (notification.actionType === ENotificationActionType.submissionCollaborationAssignment) {
+        const collaborationDate = objectData as IPermitCollaborationNotificationObjectData
+        return [
+          {
+            text: t("ui.show"),
+            href: `/permit-applications/${collaborationDate.permitApplicationId}/edit`,
```

Comment:
misleading variable name, why would a date have a permitApplicatonId?

Reply:
You probably meant data

Reply:
Yeah mistake